.azure/pipelines/ci.yml

```diff
@@ -626,7 +626,7 @@ stages:
     parameters:
       condition: ne(variables['SkipTests'], 'true')
       jobName: MacOS_Test
-      jobDisplayName: "Test: macOS 10.14"
+      jobDisplayName: "Test: macOS 10.15"
```

Comment:
nit overall: Not a fan of having platform versions in job names or display names